package.json

```diff
@@ -17,54 +17,55 @@
     "rename_bin": "mv bin/ts-react-toolbox.js bin/ts-react-toolbox",
     "release": "npm version patch && git push --tags && git push && npm publish",
     "clean": "rm -rf src/**/*.js",
-    "prepublishOnly": "yarn clean && yarn build && yarn rename_bin"
+    "prepublishOnly": "yarn clean && yarn build && yarn rename_bin",
+    "test": "echo \"no tests\""
```

Comment:
I am not sure, but travis now complains without it. I am sure there is a better way to shut it up?

Reply:
That's fine, a bit of no test shame doesn't hurt